### PR TITLE
Added set of commands needed by set up ARCHER's environment to compil…

### PR DESCRIPTION
**Pull request by raquel-ucl**
…e HJCFIT.

### CODE REVIEW

File: utils/loadArcherModules [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# Swig is needed by the python bindings
+module load swig
+
+# Load Python3
+module load anaconda-compute/2.2.0-python3
+
+# Python tests are run with behave
+# Behave won't work unless we add its install location to $PATH
+pip install behave
```
Comment (jenshnielsen) [inline utils/loadArcherModules]: This will nede a `--user` flag

File: utils/loadArcherModules [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export PATH=~/.local/bin:$PATH
+
+# Use GNU compiler suite environment
+module unload PrgEnv-cray
+module load PrgEnv-gnu/5.2.56
```
Comment (jenshnielsen) [inline utils/loadArcherModules]: Do this before installing python modules to make sure any python modules are build with gcc

**Comment by jenshnielsen** [comment]
The pull request builder is broken on legion because it doesn't set the pyversion as the main job does

**Comment by raquel-ucl** [comment]
retest this please

**Pull request merged**